app/scripts/views/sign_up.js

```diff
@@ -225,6 +227,17 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       }
       return this._isMailcheckEnabledValue;
     },
+    _wasMailcheckUseful: function () {
+      var email = this.$el.find('.email');
```

Comment:
Is there ever *not* an email address on the signup page? If the email field is always there, you could just do:

```js
var emailValue = this.getElementValue('.email');
if (emailValue) {
  ...

```